wordpress-mobile/WordPress-iOS
# Fix Blogging Reminders flow when called from Site Settings

Fixes #NA
This is the same as #16784 except it targets `17.7`

To test:
Make sure you have more than one site with no blogging reminders set (or delete and re-install the app)
Test on iPhone and iPad

- Build/run and select a site where you had not previously seen the Blogging Reminders flow
- Go to "Site Settings" then tap "Blogging Reminders" and make sure the prologue screens shows up
- Dismiss the flow (by completing the flow or dismissing any of the screens)
- Tap again on "Blogging Reminders" and verify that this time the prologue is not presented and the flow starts from the settings screen
- Publish a post (from the editor or from the draft list) and verify that the Blogging Reminders flow is not presented after publishing
- Select another site where you had not seen the Blogging Reminders yet
- Publish a post and make sure the Blogging Reminders flow starts, and the post publishing prologue is the first screen

this targets the frozen branch so that we could apply all the fixes to Blogging Reminders in `17.7`; cc @mokagio @startuptester 
## Regression Notes
1. Potential unintended areas of impact
None. This change only configures the starting point of the Blogging Reminders flow

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Test that the blogging reminders flow works as expected in all the use cases.

3. What automated tests I added (or what prevented me from doing so)
none
PR submission checklist:
